README.md

```diff
@@ -16,27 +16,24 @@ arti-ios   arti-android
        arti
 ```
        
-`Arti-rest` provides a synchronous interface to send REST requests over tor,
-and to receive the answers.
-It also does the TLS handling of the connection.
-The asynchronous part needs to be done by the Android and iOS library.
+`Arti-rest` provides a synchronous interface to send REST requests over tor, and
+to receive the answers. It also does the TLS handling of the connection. The
+asynchronous part needs to be done by the Android and iOS library.
 
 ## Reliability
 
 Even though we tried our best to produce good code, this should not be used in a
 mission critical piece of software without proper consideration of the following:
 
-- arti itself mentions it's not ready for prime-time. In their 
-  [README](https://gitlab.torproject.org/tpo/core/arti/-/blob/main/README.md), they write 
-  **you should probably not use Arti in production**
-- the current code implements caching of the tor-directory listing to avoid having
-  to download some megabytes of node descriptors every so often. Again, given the right
-  circumstances, this can be effective and secure enough. Under other circumstances
-  this is something you don't want to do.
+- arti itself mentions it's not ready for prime-time since it does not implement all protections that the stock Tor client does, see their 
+  [README](https://gitlab.torproject.org/tpo/core/arti/-/blob/main/README.md). In particular, this might mean that you should not use Arti in production.
+- arti-rest uses a customized consensus to reduce bandwidth cost for infrequent use of arti-rest. As a result, the consensus is not signed by the default Tor directory authorities, but instead by the creator of the app. This approach in general is not secure. However, since the app developers already determines which code is loaded (and could therefore disable consensus validation altogether, or deanonymize users directly), it is likely acceptable in the case of mobile applications that directly include arti-rest.
```

Comment:
```suggestion
- arti-rest uses a customized consensus to reduce bandwidth cost for infrequent use of arti-rest. As a result, the consensus is not signed by the default Tor directory authorities, but instead by the creator of the app. This approach in general is not secure. However, since the app developer already determines which code is loaded (and could therefore disable consensus validation altogether, or deanonymize users directly), it is likely acceptable in the case of mobile applications that directly include arti-rest.
```